ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsUserManagement.scala

```diff
@@ -347,7 +346,7 @@ private[backend] trait StorageBackendTestsUserManagement
       tested.getUsersOrderedById(
         maxResults = 10,
         fromExcl = None,
-        filter = IdentityProviderIdFilter.All,
+        identityProviderId = IdentityProviderId.Default,
```

Comment:
all tests use the default identity provider. Could that lead to a missed bug?

Reply:
Ah yes, this should be covered, thank you for noticing it!

Reply:
in fact I realised its a miss of previously merged PR :)